### DISCUSSION TITLE
Validation data

### DISCUSSION THREAD

**Discussion by sharaf81995** [firemodels/fds#13093] [General]
hello everyone,

Can you please provide experimental data for the transition from surface fire to crown fire, using any appropriate scale? This data will assist me in my validation process.

Regards,
Sharaf

**Comment by mcgratta** [comment]
Take a look at Section 3.14, Crown Fires, in the latest FDS Validation Guide.

**Reply by sharaf81995** [reply]
Thanks Kevin
